VideoLocker/res/layout/activity_course_base.xml

```diff
@@ -101,65 +101,62 @@
 
             <LinearLayout
                 android:id="@+id/last_access_bar"
-                android:orientation="vertical"
                 style="@style/last_access_bar_style"
-                android:visibility="gone">
```

Comment:
Sorry, I was looking at the logic in the LastAccessManager, and it seemed that there was no checking before calling the showLastAccessedView() method, but now I see that this method performs the checks itself.
> On 29-Sep-2015, at 2:37 pm, Mi@N Kh@LiD <notifications@github.com> wrote:
> 
> In VideoLocker/res/layout/activity_course_base.xml <https://github.com/edx/edx-app-android/pull/374#discussion_r40651948>:
> 
> >                  style="@style/last_access_bar_style"
> > -                android:visibility="gone">
> Are you using the user staff ? cuz he's been to everywhere  
> Otherwise, last time I checked after making a new user, Last access bar didnt show up in a course until I visited a section.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/edx/edx-app-android/pull/374/files#r40651948>.
>